index.js

```diff
@@ -156,6 +182,27 @@ app.on('ready', () => {
   }
 });
 
+// Put notes opened in a new window into focus mode
+ipcMain.on('note-frame-loaded', event => {
+  if (event.sender !== mainWindow.webContents) {
+    event.sender.send('focus-mode');
```

Comment:
Instead of creating a new note window in `focus-mode`, which can be easily untoggled by the user, we can create a new event, e.g `new-window`, that once fired both the sidebar and notes-list are completely removed from the UI using simple `css`, thus achieving a more native-feeling experience:

```js
// Changes on `index.js`
ipcMain.on('note-frame-loaded', event => {
  if (event.sender !== mainWindow.webContents) {
    event.sender.send('new-window');
  }
});
```

```js
// Addition to `browser.js`
// Activates the appropriate class names
ipc.on('new-window', () => {  document.documentElement.classList.toggle('new-window', true);}); 
```

```css
/* Additions to `style/browser.css` */
div#tusk-new-window { /* Minor positioning tweak for the new button */
  margin-right: 8px;
}
.GJDCG5CBNB { /* Also, minor positioning tweak for the new button */
  display: none !important;
}
html.new-window div#gwt-debug-sidebar, html.new-window div#gwt-debug-notesListView, html.new-window div#tusk-new-window { /* Hides both the sidebar & notes-list */
  display: none !important;
}
html.new-window div#gwt-debug-stage, html.new-window div#gwt-debug-NoteView-root { /* Removes the left margin left behind by the hidden sidebar */
  margin-left: 0px !important;
}
```